### DISCUSSION TITLE
Does Alpine have anything like React Fragment?

### DISCUSSION THREAD

**Discussion by adamjohnson** [alpinejs/alpine#989] [1. Help] [answered]
I'm making a table where I can't just insert an extra `<div>` inside my template tag. Does Alpine have anything like [React Fragment](https://reactjs.org/docs/fragments.html)? Or some other way around this?

Here's a [Codepen of a Table](https://codepen.io/serkaneken/pen/RwREXXp) with Alpine. If you inspect it, you'll notice it has several repeated `<tbody>` elements. There should be only one `<tbody>` element.

Thanks in advance for the help, team. 👍

**Comment by KevinBatdorf** [comment, marked as answer]
There isn't, but having multiple `<tbody>` elements is valid HTML and might be more correct here anyway since you are grouping those two rows as one "object", so to speak.

spec: https://html.spec.whatwg.org/multipage/tables.html#the-table-element

**Reply by adamjohnson** [reply]
Thanks for your prompt response, Kevin! I appreciate it. Also TIL multiple `<tbody>` elements isn't necessarily incorrect. Interesting.

I'm working on another table that is a bit more involved and having a `<tbody>` element wrap almost every `<tr>` doesn't make sense semantically. Maybe I'll put in an issue for a feature request. Thanks again for your help!
